nixos/modules/services/desktops/geoclue2.nix

```diff
@@ -266,5 +266,5 @@ in
       } // mapAttrs' appConfigToINICompatible cfg.appConfig);
   };
 
-  meta.maintainers = with lib.maintainers; [ worldofpeace ];
+  meta.maintainers = with lib.maintainers; [ ];
```

Comment:
These services should probably get maintainers from the package.

Reply:
I see @7c6f434c: are you OK with that?